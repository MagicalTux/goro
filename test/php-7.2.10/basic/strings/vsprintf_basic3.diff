     *** Testing vsprintf() : basic functionality - using float format ***
002- string(9) "11.110000"
003- string(9) "11.110000"
004- string(19) "11.110000 22.220000"
005- string(19) "11.110000 22.220000"
006- string(29) "11.110000 22.220000 33.330000"
007- string(29) "11.110000 22.220000 33.330000"
002+ string(17) "11.11000000000000"
003+ string(17) "11.11000000000000"
004+ string(35) "11.11000000000000 22.22000000000000"
005+ string(35) "11.11000000000000 22.22000000000000"
006+ string(53) "11.11000000000000 22.22000000000000 33.33000000000000"
007+ string(53) "11.11000000000000 22.22000000000000 33.33000000000000"
     Done
