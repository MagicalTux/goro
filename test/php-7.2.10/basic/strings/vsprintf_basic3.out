*** Testing vsprintf() : basic functionality - using float format ***
string(17) "11.11000000000000"
string(17) "11.11000000000000"
string(35) "11.11000000000000 22.22000000000000"
string(35) "11.11000000000000 22.22000000000000"
string(53) "11.11000000000000 22.22000000000000 33.33000000000000"
string(53) "11.11000000000000 22.22000000000000 33.33000000000000"
Done